src/Symfony/Contracts/HttpClient/ChunkInterface.php

```diff
@@ -47,6 +47,17 @@ public function isFirst(): bool;
      */
     public function isLast(): bool;
 
+    /**
+     * Tells when an 1xx status code was just received.
+     *
+     * @param array|null &$headers Set by reference to the headers of the informational response
+     *
+     * @return int The status code of the informational response or 0 if the response is not informational
+     *
+     * @throws TransportExceptionInterface on a network error or when the idle timeout is reached
+     */
+    public function isInformational(?array &$headers = []): int;
```

Comment:
I find the naming a bit confusing. I normally would expect a method with an `is` prefix to return a boolean.

Reply:
You can use it as if it returned a book + this name is consistent with the other methods, see the full interface. I didn't have any better idea...

Reply:
Other methods in the same interface like `isTimeout()`, `isFirst()` and `isLast()` will return a `bool` value, it's confusing that this method named `isInformational()` wouldn't, and it will return an `int` value so what about naming it `getInformational()`, `getInformationalStatus()` or `getInformationalStatusCode()` instead?

Reply:
I've just renamed the method `getInformationalStatus`